tests/MenderAPI/admission.py

```diff
@@ -60,8 +60,9 @@ def get_devices_status(self, status=None):
 
 
     def set_device_status(self, device_id, status):
-        r = self.s.put(self.admission_base_path + "devices/%s/status" % device_id,
-                         headers={'Content-Type': 'application/json'},
+        r = requests.put(self.admission_base_path + "devices/%s/status" % device_id,
+                         verify=False,
+                         headers={'Content-Type': 'application/json', self.auth_header.keys()[0]: self.auth_header.values()[0]},
                          data=json.dumps({"status": status}))
```

Comment:
I would instead do:

```
headers = dict(self.auth_header)
headers['Content-Type'] = 'application/json'
r = requests.put(self.admission_base_path + "devices/%s/status" % device_id,
                         verify=False,
                         headers=headers,
                         data=json.dumps({"status": status}))
```

This will save us a lot of headache later if we ever add any fields to the `auth_header`.